src/librustc_mir/const_eval/machine.rs

```diff
@@ -177,7 +178,7 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter {
 
     type MemoryMap = FxHashMap<AllocId, (MemoryKind<!>, Allocation)>;
 
-    const STATIC_KIND: Option<!> = None; // no copying of statics allowed
+    const GLOBAL_KIND: Option<!> = None; // no copying of globals allowed
```

Comment:
What does the comment mean?

Reply:
See [here](https://github.com/rust-lang/rust/pull/70241/files#diff-409e97970946f7c5fa9ad24685c617b3R112).

Reply:
Ah, "from `tcx`" was the missing part, at a glance this looks like it's talking about `let x = STATIC;` which confused me.